python/doc/examples/probabilistic_modeling/distributions/plot_pointConditionalDistribution.py

```diff
@@ -0,0 +1,255 @@
+"""
```

Comment:
all examples are lowercase:
plot_point_conditional_distribution.py

Reply:
ok! do I have to write:
git mv plot_pointConditionalDistribution plot_point_conditional_distribution.py
or
git rename plot_pointConditionalDistribution plot_point_conditional_distribution.py ?

Reply:
yes git rm

Reply:
I did it!